csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -768,16 +760,63 @@ abstract class TranslatedVariableAccess extends TranslatedNonConstantExpr {
     result = getTranslatedExpr(expr.(QualifiableExpr).getQualifier())
   }
 
-  override Instruction getResult() { result = this.getInstruction(OnlyInstructionTag()) }
+  override Instruction getResult() {
+    if this.needsExtraLoad()
+    then result = this.getInstruction(LoadTag())
+    else result = this.getInstruction(AddressTag())
```

Comment:
How can `this.getInstruction(AddressTag())` return anything when the override of `hasInstruction` below only returns tuples with `tag = LoadTag()`?

Reply:
The two classes that implement the `TranslatedVariableAccess` abstract class do have instructions with tag `AddressTag()`.

Reply:
OK.